sopel_modules/rat_board.py

```diff
@@ -2037,3 +2037,21 @@ def cmd_invalid(bot, trigger, caseid):
 
     except:
         bot.reply('Couldn\'t find a case with id ' + str(caseid) + ' or other APIError')
+
+
+@commands('lang', 'language')
+@parameterize('rw', usage='<case number or name> <language code>')
+@require_permission(Permissions.Rat)
+def cmd_lang(bot, trigger, case, lang):
+    """
+    Sets a case's language
+    """
+    lang = lang.lower()
+    if lang in ["en", "cs", "nl", "fr", "de", "no", "pt", "ru", "es", "sv"]:
+        with bot.memory['ratbot']['board'].change(case):
+            case.data.update({'langID': lang})
+
+        save_case_later(bot, case, forceFull=True)
+        bot.say('Language on case {case.client_name} changed to {lang}.'.format(case=case, lang=lang))
+    else:
+        bot.say('Unrecognized language code: ' + lang)
```

Comment:
Now i can't find a technical problem with this but my preference is to avoid string concatenation in favor of `str.format` and `fstring`. 

Due to the python environment the bot runs on we are limited to python3.5.x language, thus `fstring` isn't an option.

Could you please change this to also use `str.format`? keeps things consistent

Reply:
Usually I might agree with you, but in this case using `str.format` over simple concatenation would hurt both readability and simplicity in my eyes. As for consistency, the rest of the code uses both options as well.
If you still want me to change it, this isn't something I'll bicker about, but I'd rather not.

Reply:
I was expressing an opinion on the string concat, Mecha2 is written really inconsistently at the best of times. Methinks you can leave that as-is, you are right to say its easier to read. As long as we don't end up with
 ```py
"foo"+bar+"="+bar.value
```
 or worse. For something as simple as this i'm not interested in perusing it.

Reply:
Agreed. For more complex things format (or % if it weren't deprecated) is instrumental